src/ProductsXmlFeed.php

```diff
@@ -414,7 +414,16 @@ private static function get_taxonomies( $product_id ) {
 	}
 
 	/**
-	 * Fetch shipping object.
+	 * Get locale currency decimals
+	 */
+	private static function get_currency_decimals() {
```

Comment:
This won't produce the desired output if for example my base country is US and my currency is JPY.
We need to get the decimals from that array based on the currency returned by `get_woocommerce_currency()`